main.go

```diff
@@ -44,6 +47,8 @@ func main() {
 		Default(defaultEventQueueSize).IntVar(&config.EventQueueSize)
 	kingpin.Flag("base-role-arn", "Base Role ARN. If not defined it will be autodiscovered from EC2 Metadata.").
 		StringVar(&config.BaseRoleARN)
+	kingpin.Flag("assume-role-arn", "Assume Role ARN can be specified to assume a role at start-up which is used for further assuming other roles managed by the controller.").
```

Comment:
This flag will benefit from more in-depth treatment in the _README.md_ file (with a worked example showing how three or more IAM roles relate), but we can save that for a later patch.

Reply:
Thanks a lot for your feedback!

I have added documentation to the README.